app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,20 @@
+.site-main {
+  padding: 2em 0; // should this be in here?
+}
+
+.site-main--simple {
+  .container {
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.